src/components/icon/index.js

```diff
@@ -12,6 +12,8 @@ const Icon = (props) => {
     color,
   } = props;
 
+  if (name === undefined) return null;
```

Comment:
name props is a required item, so it can't be undefined, can it?

Reply:
However, anyone use something like this: `<Icon />`. In this case, we don't render anything and prop-types validation throw an error in console. This line of code has been written to pass one of the unit tests.